packages/sensors/example/lib/snake.dart

```diff
@@ -0,0 +1,114 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:math' as math;
+
+import 'package:flutter/material.dart';
+import 'package:sensors/sensors.dart';
+
+class Snake extends StatefulWidget {
+  final int rows;
+  final int columns;
+  final double offsetSize;
+  Snake({this.rows = 20, this.columns = 20, this.offsetSize = 10.0});
```

Comment:
Should probably assert that these numbers are reasonable. E.g.
```dart
assert(10 <= rows);
assert(10 <= columns);
assert(5.0 <= offsetSize);
```
Cf. how we initialize `snakeLength` below.